docs/keto/modeling/learn-the-concepts.mdx

```diff
@@ -0,0 +1,295 @@
+---
+id: learn-the-concepts
+title: Create your first permission model with OPL
+sidebar_label: Create your first permission model
+---
+
+```mdx-code-block
+import Mermaid from "@site/src/theme/Mermaid"
+```
+
+This section guides you through creating your first permission model using the Ory Permission Language (OPL).
+
+## What is a permission model
+
+Permission checks are answered based on data in the database, for example Bob is owner of document X, as well as the permission
+model, for example all owners of a document can view it.
+
+When you ask Ory Permissions if Bob can view document X, it will check if Bob is owner of the document X. The permission model
+tells Ory Permissions what to check in the database.
```

Comment:
I think the permission model is rather the collection of all namespaces and permission rules. What you described is rather a single permission rule. Also, I would suggest to use the new relationship string format with syntax highlighting:

```suggestion
Permission checks are answered based on data in the database, for example <code class="language-keto-relationships">User:Bob is owner of Document:X</code>, as well as permission rules
like "all owners of a document can view it".

When you ask Ory Permissions <code class="language-keto-relationships">is User:Bob allowed to view on Document:X</code>, it will check if Bob is owner of the document X. The permission model
tells Ory Permissions what to check in the database.
```

Reply:
btw I figured out the inline syntax highlighting does not work...